practice-one/src/assets/styles/sections/footer.css

```diff
@@ -0,0 +1,71 @@
+.footer-background {
+  background-color: var(--black-color);
+}
+
+.footer-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 105px;
+  padding-bottom: 91px;
+}
+
+.footer-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  color: var(--white-color);
+}
+
+.footer-list .description {
+  opacity: 0.5;
+  color: inherit;
+}
+
+.footer-list .menu-item {
+  color: inherit;
+}
+
+.footer-item {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.item-title {
+  font-family: var(--primary-font);
+  font-weight: var(--font-weight-regular);
+  font-size: var(--font-size-lg);
+  letter-spacing: 1px;
+  color: inherit;
+  text-transform: uppercase;
+}
+
+.address {
+  display: flex;
+  flex-direction: row;
+  gap: 54px;
+}
+
+.address .description {
+  max-width: 133px;
+  max-height: 77px;
+}
+
+.socials {
+  display: flex;
+  gap: 43px;
+}
+.social-icon {
+  opacity: 0.5;
+}
+
+.footer-separator {
```

Comment:
What is the purpose of this class?

Reply:
This class allow me to create a separator between two parts of footer section 

![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/f7a1d89a-e54f-4f75-b942-722966c1f417)